noir-projects/aztec-nr/value-note/src/balance_utils.nr

```diff
@@ -3,10 +3,12 @@ use dep::aztec::{
     context::UtilityContext, note::note_viewer_options::NoteViewerOptions, state_vars::PrivateSet,
 };
 
+// DEPRECATED: Having this functionality in the value note package is incorrect and will be removed in the future.
```

Comment:
If we would want this then I think it should be associated with state vars and not with a note.